enterprise/backend/src/metabase_enterprise/task/cache.clj

```diff
@@ -125,6 +142,13 @@
             parameterized-queries (t2/select :model/Query (duration-queries-to-rerun-honeysql cache-configs true))]
         (concat base-queries (select-parameterized-queries parameterized-queries))))))
 
+(defn- clear-duration-caches!
+  "Deletes any existing cache entries for queries that we are about to re-run, so that subsequent tasks don't also try
+  to re-run them before the cache has been refreshed."
+  [queries]
+  (t2/delete! :model/QueryCache
+              {:where [:in :query_hash (map :cache-hash queries)]}))
```

Comment:
I think the following would work too:
```suggestion
  (t2/delete! :model/QueryCache :query_hash [:in (map :cache-hash queries)]))
```

Looking at the implementation, it's not obvious that has only to do with duration caches, so I wonder if the name is correct.